From 1cf04aef5dcfec540b71f5ccb2b9702d81ca0184 Mon Sep 17 00:00:00 2001
From: Iago Estrela <IagoFilipe.EstrelaBarros@windriver.com>
Date: Wed, 12 Jan 2022 17:46:10 -0300
Subject: [PATCH] Replace deprecated Nova VNC configurations

Replace deprecated opts of Nova VNC Server configurations.

Signed-off-by: Iago Estrela <IagoFilipe.EstrelaBarros@windriver.com>
[ upversioned openstack-helm base commit ]
Signed-off-by: Thales Elero Cervi <thaleselero.cervi@windriver.com>
Change-Id: I7103d2ae77bec442790a0bbed3aac53fe1e7641e
---
 nova/templates/bin/_nova-console-proxy-init.sh.tpl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nova/templates/bin/_nova-console-proxy-init.sh.tpl b/nova/templates/bin/_nova-console-proxy-init.sh.tpl
index f3eb9f31..f3522c9c 100644
--- a/nova/templates/bin/_nova-console-proxy-init.sh.tpl
+++ b/nova/templates/bin/_nova-console-proxy-init.sh.tpl
@@ -19,7 +19,7 @@ set -ex
 console_kind="{{- .Values.console.console_kind -}}"
 
 if [ "${console_kind}" == "novnc" ] ; then
-    client_address="{{- .Values.conf.nova.vnc.vncserver_proxyclient_address -}}"
+    client_address="{{- .Values.conf.nova.vnc.server_proxyclient_address -}}"
     client_interface="{{- .Values.console.novnc.vncproxy.vncserver_proxyclient_interface -}}"
     listen_ip="{{- .Values.conf.nova.vnc.server_listen -}}"
 elif [ "${console_kind}" == "spice" ] ; then
@@ -49,7 +49,7 @@ fi
 if [ "${console_kind}" == "novnc" ] ; then
 cat <<EOF>/tmp/pod-shared/nova-vnc.ini
 [vnc]
-vncserver_proxyclient_address = $client_address
+server_proxyclient_address = $client_address
 server_listen = $listen_ip
 novncproxy_host = $listen_ip
 EOF
-- 
2.25.1

